.github/workflows/yamllint-check.yml

```diff
@@ -6,6 +6,7 @@ on:
   pull_request:
     types:
       - opened
+      - edited
```

Comment:
@ShakedAviv50313 try deleting these three lines. Only with `pull_request` should be enough